src/test/java/com/wikia/webdriver/common/core/api/CuratedContent.java

```diff
@@ -0,0 +1,40 @@
+package com.wikia.webdriver.common.core.api;
+
+import com.wikia.webdriver.common.core.XMLReader;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.io.File;
+
+public class CuratedContent extends ApiCall {
+
+  private CuratedContent() {
+  }
+
+  private static String secret;
+  private static String baseURL;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this variable to comply with Java Code Conventions. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1213)